crates/solidity/outputs/typescript/lib/.github/workflows/CI.yml

```diff
@@ -1,27 +1,26 @@
-# yamllint disable-file
```

Comment:
I suggest also keeping the ignore here, and undoing the change in `.yamllint.yml`.

Reply:
done